src/sentry/models/groupowner.py

```diff
@@ -1,24 +1,34 @@
 from collections import defaultdict
 from enum import Enum
+from typing import List, TypedDict
 
 from django.conf import settings
 from django.db import models
 from django.utils import timezone
 
+from sentry import features
 from sentry.db.models import FlexibleForeignKey, Model
 
 
 class GroupOwnerType(Enum):
     SUSPECT_COMMIT = 0
     OWNERSHIP_RULE = 1
+    RELEASE_COMMIT = 2
 
 
 GROUP_OWNER_TYPE = {
     GroupOwnerType.SUSPECT_COMMIT: "suspectCommit",
     GroupOwnerType.OWNERSHIP_RULE: "ownershipRule",
+    GroupOwnerType.RELEASE_COMMIT: "releaseCommit",
```

Comment:
Not sure how I feel about prematurely adding these here. Might give off the impression we;re storing these in the db when we're not.

Reply:
good point, removed